modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -270,6 +271,7 @@ The value can be a plain number or a byte units string, e.g. `128k`, `1m`.
 * The first line must contain the header.
 * Multiple data sources like these can be specified in one import, where each data source has its own header.
 * Files can also be specified using regular expressions.
+* As of Neo4j 5.19, files can also be located in S3 URIs.
```

Comment:
This could be added in the same note as `--nodes` and we can have one example for both.